### PR TITLE
fix: edge case of tests marked as obsolte

### PR DESCRIPTION
**Pull request by gkampitakis**
…eliable

### CODE REVIEW

File: snaps/clean.go [hunk 1]
```diff
@@ -233,23 +190,14 @@ func occurrences(tests map[string]int) Set {
...
-	e.g
-	func TestParallel (t *testing.T) {
-		snaps.Skip()
-		...
-	}
-	Then every "child" test should be skipped
-*/
-func testSkipped(testID string) bool {
-	isSkipped := false
+func parseRunFlag() (runOnly string) {
```
Comment (gkampitakis) [inline snaps/clean.go:193]: Write unit test for this
